src/app/components/sidebar/filters/price/Price.test.tsx

```diff
@@ -0,0 +1,24 @@
+import { Price } from './Price';
+import { render } from 'tests';
+
+describe('Price component', () => {
+  it('renders price range inputs correctly', () => {
+    const { getByText, queryByTestId } = render(
+      <Price onInputChange={() => {}} minPrice={undefined} maxPrice={undefined} />,
+    );
+
+    expect(getByText('-')).toBeInTheDocument();
+    expect(getByText('from')).toBeInTheDocument();
+    expect(getByText('to')).toBeInTheDocument();
+    expect(queryByTestId('checkbox')).not.toBeInTheDocument();
+  });
+
+  it('renders price in input, and checkbox', () => {
```

Comment:
Mhm, ale kiedy ma być to wyrenderowane? Dobrze byłoby to dodać do opisu scenariusza